development/build/index.js

```diff
@@ -177,12 +177,6 @@ function parseArgv() {
     throw new Error('MetaMask build: No entry task specified.');
   }
 
-  // The entry task should never be prefixed with '-' or '--', so that is easily
-  // distinguishable from other arguments.
-  if (entryTask.startsWith('-')) {
-    throw new Error(`MetaMask build: invalid entry task: "${entryTask}"`);
-  }
-
```

Comment:
Redundant since we started grabbing positionals from the `minimist` `argv` underbar property. It doesn't put `-`-prefixed arguments in that array.